src/components/pill-selector-dropdown/PillSelector.js

```diff
@@ -289,4 +304,11 @@ class PillSelector extends React.Component<Props, State> {
     }
 }
 
+export { PillSelectorBase };
+
+const PillSelector = React.forwardRef<Config, PillSelectorBase>((props: Config, ref: React.Ref<any>) => (
+    <PillSelectorBase {...props} innerRef={ref} />
+));
+PillSelector.displayName = 'PillSelector';
```

Comment:
is the `displayName` missing if we remove this line?

it'd be great if we could do this:
```
class PillSelector extends [...]

[...]

export { PillSelector as PillSelectorBase };
export default React.forwardRef[...]
    <PillSelector {...props} innerRef={ref} />
));
```
since that's the common pattern for this repo

Reply:
`displayName` will be missing without it. Otherwise the snapshots will show something like `<ForwardRef>` instead of `<PillSelector>`

Reply:
This will break existing tests that rely on finding a component by displayName. 

`expect(wrapper.is('SearchForm')).toBe(true);`

Reply:
gotcha, makes sense. thanks for checking!